README.md

```diff
@@ -1,3 +1,7 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+Northcoders News API is an API for the purpose of accessing application data programmatically. The intention here is to mimic the building of a real world backend service (such as Reddit) which should provide this information to the front end architecture.
+
+### Enivronment variables
+Create two .env files: .env.test and .env.development. Into each, add PGDATABASE=<database_name_here>, with the correct database name for that environment. 
+.env files must be .gitignored
```

Comment:
Cool, clear instructions 👍 
If you add backticks \` around words you can make them look `all codey`, which could be useful to use here.